src/Umbraco.ModelsBuilder/Umbraco/PureLiveModelFactory.cs

```diff
@@ -300,19 +328,12 @@ internal Infos EnsureModels()
 
         private Assembly GetModelsAssembly(bool forceRebuild)
         {
-            var modelsDirectory = UmbracoConfig.For.ModelsBuilder().ModelsDirectory;
+            var modelsDirectory = _config.ModelsDirectory;
             if (!Directory.Exists(modelsDirectory))
                 Directory.CreateDirectory(modelsDirectory);
 
-            // must filter out *.generated.cs because we haven't deleted them yet!
-            var ourFiles = Directory.Exists(modelsDirectory)
```

Comment:
I wonder why we are not using this as part of the hash any longer? Seems like an odd change since this is most likely how it is working in the original MB

Reply:
I don't believe this should have been removed, I will re-implement these checks in my new branch.

Reply:
Actually, on second thought, the reason this is removed is because it's not really including any files since the only files that will exist in this folder that do not end with .generated.cs is just the hash file. So i guess inluding these files is useless. Perhaps this is for when DLL mode is enabled, but it seems not relevant for either purelive appdata or liveappdata.